NewSpring/ApollosProject
# Updates Fragment Types

## DESCRIPTION

Not sure why this changed but it did for me.

### What does this PR do, or why is it needed?

### How do I test this PR?

Run `yarn start` and make sure you don't have any uncommited changes pop out.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
